resources/settings.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings>
-   <setting label=30053 id="disabled" type="bool" value="" />
-   <setting id="last_update" value="" visible="false" />
+    <setting id="last_update" value="" visible="false" />
+    <setting id="version" value="" visible="false" />
+    <category label="30900"> <!-- Troubleshooting -->
+        <setting label="30903" help="30904" type="bool" id="disabled" value="false"/>
+        <setting label="30904" type="text" enable="false"/> <!-- disabled_warning -->
```

Comment:
The #30904 string is too long to show on one line in the settings dialog, it breaks at `playback ma...`

Reply:
It worked for my setup at home and my celphone, but I was worried this, yes.
The real solution is to use the new v18 settings syntax which can show help statements.